src/components/CompetitionPage/CompetitionPage.tsx

```diff
@@ -48,11 +48,20 @@ export const CompetitionPage: FC<CompetitionPageProps> = ({
 
   return (
     <Stack gap={3}>
-      {who_can_participate && <Typography variant="body1">Súťaž je určená pre {who_can_participate}</Typography>}
+      {who_can_participate && (
+        <Typography variant="body1">
+          Súťaž je určená pre {who_can_participate}.{description && ` ${description}`}
+        </Typography>
+      )}
 
-      {description && <Typography variant="body1">{description}</Typography>}
-
-      <Stack alignSelf="center">
+      <Stack
+        direction="row"
+        sx={{
+          mt: 0.5,
+          justifyContent: 'end',
+          alignItems: 'end',
```

Comment:
basically staci:
```
        sx={{
          mt: 0.5,
          alignItems: 'end',
```
`alignItems` zarovnava v "cross" osi - ked mame Stack, je defaultne vertikalny, a `alignItems` to da na koniec horizontalne.
alternativa by bola:
```
        direction="row"
        sx={{
          mt: 0.5,
          justifyContent: 'end',
```
`justifyContent` zarovnava v osi sadzania

Reply:
Použil som to prvé